networking/src/Models/ITeamAnswerContent.ts

```diff
@@ -0,0 +1,29 @@
+import { GameSessionState } from "../AWSMobileApi";
+enum AnswerType {
+  MULTICHOICE,
+  TEXT,
+  NUMBER,
+  FORMULA,
+  NULL
+}
+
+export interface INormAnswerSubObj {
+  value: string | number;
+  type: AnswerType;
+}
+
+export interface INormAnswer {
+  raw: string;
+  norm?: INormAnswerSubObj[];
+}
+
```

Comment:
This has been updated per: 
[[Host/Play/Networking] - Short Answer Response - Adjusts Answer Handling #858](https://github.com/rightoneducation/righton-app/pull/858)